ideascube/conf/base.py

```diff
@@ -302,3 +302,6 @@
 
 SESSION_COOKIE_AGE = 60 * 60  # Members must be logged out after one hour
 BACKUP_FORMAT = 'zip'  # One of 'zip', 'tar', 'bztar', 'gztar'
+
+CATALOG_CACHE_BASE_DIR = '/var/cache/ideascube/catalog'
```

Comment:
You're right, I forgot that.

This is now done in the latest push to this branch.